### DISCUSSION TITLE
Can this work for a subdomain only?

### DISCUSSION THREAD

**Discussion by cben0ist** [velsa/notehost#21] [General]
Hi,
I am trying to point a subdomain to a notion page but don't want to change my current site and www.
Can I do that with this tool?
Thank you!

**Comment by james-tindal** [comment]
In site config, set domain to subdomain.domain.com.
In wrangler.toml, set pattern to subdomain.domain.com

**Comment by DudeThatsErin** [comment]
I created detailed instructions on how to do this and the differences for setting up for a subdomain are listed in the FAQs: https://dudethatserin.notion.site/NoteHost-982d31fcc8dd4799a18efcb074b0e63c
